### ISSUE TITLE
Plain text between html tags aren't transliterated

### ISSUE THREAD

**Issue by fontanon** [andalugeeks/andaluh-browser-extension#3]
The whole plain text of an element is escaped from translation when any HTML tags are found (see line `42` for `src/contentScript.js`):

```javascript
40      for (const text of texts) {
41        // If innerText contains any HTML tags, ignore it
42        if (text.innerHTML.indexOf('<') === -1) {
43          // Translate the text
44          const translated = EPA.transcript(text.innerHTML, 'ç', 'h', true);
45          // Update the text
46          text.innerHTML = translated;
47          text.classList.add('andaluh-translated');
48        }
49      }
50    }
```

Hence, text elements like the following below aren't translated: 

```html
<p>
    Texto para ser transcrito 
    <a href="https://andaluh.es"> pero como hay un inner link </a> 
    pues ya no se transcribe
</p>
```

**Comment by JoseCarlosGarcia95** [comment]
It is done on purpose, if I don't escape the HTML, it breaks the web design, since it tries to transcribe all the tags, classes, etc, causing unexpected behaviors. For that reason, I have a whitelist of HTML tags, in line 27.

**Comment by fontanon** [comment]
> It is done on purpose, if I don't escape the HTML, it breaks the web design, since it tries to transcribe all the tags, classes, etc, causing unexpected behaviors. For that reason, I have a whitelist of HTML tags, in line 27.

I see. I might not have expressed the issue correctly. Please try using the plugin with any wikipedia page as: https://es.wikipedia.org/wiki/Andaluc%C3%ADa

@JoseCarlosGarcia95 please have a look at the screenshot: https://drive.google.com/file/d/1i9v5X11tYe9Zor8N3Rw6cu1IgPsziaNk/view?usp=sharing

**Comment by penyaskito** [comment]
Started https://github.com/andalugeeks/andaluh-chrome-extension/pull/4 trying to accomplish this.

**Comment by fontanon** [comment]
Closed with #5
